content/theme/minimal-bootstrap-blog.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Please remove any comments from the frontmatter

Reply:
Thanks!